controller/persistence/migrations_v1_pg.go

```diff
@@ -196,11 +197,6 @@ func migrateServicesFromPG(mtx *MigrationContext) error {
 			clusterIds = append(clusterIds, cluster.ID)
 		}
 
-		var hostedIds []string
```

Comment:
Aren't these needed for PG upgrades?

Reply:
AFAIK, hosted IDs aren't being used anywhere important yet, so I decided not to bother trying to migrate them